integration/images/volume-ownership/Dockerfile

```diff
@@ -12,7 +12,7 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 
-FROM busybox
+FROM ubuntu
```

Comment:
nec?

Reply:
"busybox now does not have nogroup, since the test needs it, switch over from busybox to ubuntu for just the volume-ownership image"

Reply:
yes this is an intentional change.